src/pages/Home.js

```diff
@@ -2,11 +2,13 @@
 
 import CreateBook from '../components/CreateBook';
 import BooksList from '../components/BooksList';
+import './home.css';
 
 export default function Home() {
```

Comment:
- [ ] I kindly suggest that you get rid of the horizontal scroll on your page in order to make your design match the Zeplin design better


<img width="960" alt="image" src="https://github.com/majdalkilany/Book-Store/assets/52934128/c38f5135-9253-4b87-8390-a98b0909944a">